xdsl/dialects/builtin.py

```diff
@@ -1037,6 +1089,11 @@ def parse_with_type(
     def print_without_type(self, printer: Printer):
         return printer.print_float_attr(self)
 
+    def print_builtin(self, printer: Printer):
+        self.print_without_type(printer)
+        printer.print_string(" : ")
+        printer.print_attribute(self.get_type())
```

Comment:
There's only 3 or 4 builtin TypedAttributes anyway, so I've inlined the logic as that seemed like the easiest way deal with it.

Reply:
Yeah, this was not really well designed in the first place I feel, so as long as it works for our examples, it should be fine.
I think we can clean this class later (one day)